data/uhc_pack/functions/lobby/update_sidebar.mcfunction

```diff
@@ -0,0 +1 @@
+execute store result score Players uhcLobby run scoreboard players get Players uhcState
```

Comment:
There is currently no logic to make this scoreboard display (no `scoreboard objectives setdisplay sidebar` command). This should be added for this to function correctly.

When adding it, the other things you'll need to consider:
* The scoreboard should be hidden when the game starts
* The pre-generation also uses a similar scoreboard, so you will need to ensure that when the pre-generation completes you set the 'Game Details' scoreboard back to display.